### ISSUE TITLE
Demand outputs 0 on initial trigger

### ISSUE THREAD

**Issue by jamshark70** [supercollider/supercollider#6010]
## Environment

* SuperCollider version: 13 (I think)
* Operating system: n/a
* Other details (Qt version, audio driver, etc.): n/a

## Steps to reproduce

```supercollider
(
a = {
	var trig = Impulse.kr(1);
	var demand = Demand.kr(trig, 0, Dseq([100, 101], inf));
	demand.poll(trig);
	Silent.ar(1);
}.play;
)

UGen(OutputProxy): 0  // not 100 ??

a.release;
```

## Expected vs. actual behavior

I'd expect Demand to start with the first value in the sequence.

It seems to output an initial 0 for one sample, then jump to the correct value.

Unsure if this is fixed since my last build (I have a show tonight and I won't rebuild SC to test until after that).

**Comment by jamshark70** [comment]
Hm, I'll retract this one for now. It may be an issue with a local version of Poll only.

This seems not to reproduce the issue (y axis ranges 0.1 to 0.5, not 0 to 0.5 if it were reproduced):

```
{ Demand.ar(Impulse.ar(440), 0, Dseries(1, 1, inf) * 0.1) }.plot;
```

**Comment by dyfer** [comment]
> It may be an issue with a local version of Poll only.

I think I can confirm this

On the 3.13.0 release (macOS) I seem to get the proper result:
```
-> Synth('temp__0' : 1000)
UGen(OutputProxy): 100
UGen(OutputProxy): 101
UGen(OutputProxy): 100
UGen(OutputProxy): 101
(...)
```
I think Poll was fixed in #5965 sometime before the final 3.13.0 release.
